src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '2.5.1'
+ModuleVersion = '2.5.2'
```

Comment:
Do we really want to change that to 2.5.2? 
Anyway, 2.5.2 is also OK for me, but 2.5.1 will look better, as skip a version looks a little weird.